vars/writeBuildInfo.groovy

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/groovy
+
+import groovy.json.JsonBuilder
+import groovy.json.JsonSlurperClassic
+import org.feedhenry.Utils
+
+def call(name, versionTxt) {
+    def utils = new Utils()
+    def buildInfoFileName = utils.getBuildInfoFileName()
+    def buildInfo = [:]
+
+    if(fileExists(buildInfoFileName)) {
```

Comment:
I guess this is just so that we can get the previous contents easily enough, even if we overwrite them here?